app/controllers/invitations_controller.rb

```diff
@@ -13,10 +13,13 @@ def new
   def create
     @invitation = current_user.invitations.build(invitation_params)
     @invitation.authorizations = get_socials
-
+    friend = @invitation.friend
+    @invitation.authorizations.each do |authorization|
+      Invitation.request(authorization.provider, current_user, friend);
+    end
```

Comment:
maybe we should do this in the ```@invitation.save``` block? we probaly dont want to send invitation if the ```@invitation``` is invalid